index.js

```diff
@@ -53,20 +53,7 @@ function notifyOutdatedVersion(releaseData) {
     });
 }
 
-function getManifestVersion(manifest) {
-    if (!manifest) {
-        manifest = chrome.runtime.getManifest();
-    }
-
-    var version = manifest.version_name;
-    if (!version) {
-        version = manifest.version;
-    }
-
-    return version;
-}
-
-checkForConfiguratorUpdates();
+if (isNW()) { checkForConfiguratorUpdates(); }
```

Comment:
The same. Does this not work under Chrome? And please, not all in the same line.

Reply:
This uses chrome.storage API which is also only available to extensions. Using localStorage would be possible, but then again the version check and update process is not possible to debug this way.